itest/src/it/itest-standalone/src/test/java/org/ops4j/pax/cdi/test/InterBundleProducesTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
```

Comment:
This not code to merge. The test case fails obviously because of the issue. It's just a test case reproducing the issue.